### ISSUE TITLE
SID on Low Pop Server

### ISSUE THREAD

**Issue by oholseven** [jasonrohrer/OneLifeData7#1146]
Will the game allow up to 4 Eves on Low Pop, considering the SID change?  If you want to Eve to your own base, it sounds like bouncing between 2 fertiles could throw you into DT.

**Comment by MooseSeven** [comment]
Tarr's original suggestion would solve a problem without introducing new ones:
"Just make it always cycle and not be completely random."

Even main server sometimes enter phases of low population. When a family struggles, males or elders sometimes /die to try to be reborn as a girl. The last fertile mother might not have many minutes left when this action is taken, and the person who tries to save the family might end up in donkey town if he doesn't reach her in time.

**Comment by jasonrohrer** [comment]
Yeah, I will patch this up.  I agree about the runner babies.  That did suck.
